src/config.rs

```diff
@@ -200,30 +231,48 @@ impl ConfigBuilder {
     }
 
     /// Set time format string to use rfc2822.
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_format_rfc2822(&mut self) -> &mut ConfigBuilder {
         self.0.time_format = TimeFormat::Rfc2822;
         self
     }
 
     /// Set time format string to use rfc3339.
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_format_rfc3339(&mut self) -> &mut ConfigBuilder {
         self.0.time_format = TimeFormat::Rfc3339;
         self
     }
 
+    /// Set offset used for logging time (default is 0)
+    #[cfg(feature = "chrono")]
+    pub fn set_time_offset(&mut self, time_offset: FixedOffset) -> &mut ConfigBuilder {
```

Comment:
As per suggestion above this function would change it's parameter to something like `impl Into<Offset>` (if `Offset` is our wrapper type).